### DISCUSSION TITLE
Connecting BudiBase to MySQL in Hostinger

### DISCUSSION THREAD

**Discussion by DevAlzion01** [Budibase/budibase#13074] [Help]
Hello everyone. I'm new to Budibase and I'm trying to connect Budibase to a SQL DB in Hostinger. Already configured the credentials. Tested it with port 3306 and with the SSH port, both username and passwords and I continue getting an error.

Any help would be appreciated.

**Comment by ConorWebb96** [comment]
Hey @DevAlzion01,

- Are you able to connect to it with a database GUI? (DB beaver, tableplus etc) 
- Is there a firewall, do you possibly need to [whitelist](https://docs.budibase.com/docs/whitelisting) Budibase's ip addresses? 
- Can you screenshot the error you are receiving?

I've connected to an external db server and everything seems to be working fine. The issue is likely to do with your configuration.
